src/manipulation.js

```diff
@@ -201,7 +201,7 @@ jQuery.extend({
 		return clone;
 	},
 
-	buildFragment: function( elems, context, scripts, selection ) {
+	buildFragment: function( elems, context, scripts, selection, ignored ) {
```

Comment:
Should we make `buildFragment` private and change it signature, 5 arguments usually considered as bad API, even for helpers. One object as an argument?